bulbs/api/views.py

```diff
@@ -356,9 +356,9 @@ def create(self, request, *args, **kwargs):
             self.post_save(self.object, created=True)
             headers = self.get_success_headers(serializer.data)
             return Response(serializer.data, status=status.HTTP_201_CREATED,
-                            headers=headers)
+                    headers=headers)
 
-        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
This looks like an indentation error. Original indentation would hit this line when IF statement on line 353 failed. As it stand now this line will never fire because of same-indent-level RETURN on line 358